tests/integration_tests/platforms.py

```diff
@@ -72,12 +72,12 @@ def launch(self):
             launch_args['instance_type'] = self.instance_type
         launch_args.update(self.launch_kwargs)
         self.instance = self.client.launch(**launch_args)
-        print('Launched instance: {}'.format(self.instance))
+        log.info('Launched instance: %s', self.instance)
 
     def destroy(self):
         self.instance.delete()
 
-    def exec(self, command):
+    def execute(self, command):
```

Comment:
@smoser It appears we can do both. These both work:
```
print(client.execute('cloud-init -v'))
print(client.execute(['/usr/bin/cloud-init', '-v']))
```

Reply:
Are you sure?

Try this:

    >>> subprocess.check_output(['echo', 'hi mom!'])
    b'hi mom!\n'
    >>> subprocess.check_output(['echo', 'hi mom!'], shell=True)
    b'\n'

I suspect your client.execute(['echo', 'hi mom!']) will do something like the second.

If it got that right, try this:

    client.execute(['echo', """Please put "2 spaces" after a period.  You'll make more $$ when you grow up. Sincerely, your 8th grade teacher"""])

Its worth getting 'exec' right once.

Reply:
actually... it might do the right thing.
[pycloudlib.execute](https://github.com/canonical/pycloudlib/blob/master/pycloudlib/instance.py#L132) uses [shell_pack](
https://github.com/canonical/pycloudlib/blob/master/pycloudlib/util.py#L80).  Hooray!

Thanks @powersj

Reply:
Confirmed. 
```
class TestExample:
    def test_example(self, client):
        print(client.execute(['echo', 'hi mom!']))
        print(client.execute(['echo', """Please put "2 spaces" after a period.  You'll make more $$ when you grow up. Sincerely, your 8th grade teacher"""]))
```

will output
```
hi mom!
Please put "2 spaces" after a period.  You'll make more $$ when you grow up. Sincerely, your 8th grade teacher
```

Reply:
@smoser I had a really good teacher :)